src/store/ordinoSlice.ts

```diff
@@ -65,16 +75,26 @@ export interface IWorkbench {
 
   data: { [key: string]: IRow };
   columnDescs: (IColumnDesc & { [key: string]: any })[];
-  // TODO: how do we store the lineup-specific column descriptions?
+  // TODO: how do we store the lineup-specific column descriptions? give an example?
```

Comment:
I am also not sure what this means, maybe related to https://app.zenhub.com/workspaces/reprovisyn-60bf3a424e19290011f57e06/issues/datavisyn/reprovisyn/238